nrclient/nrclient.go

```diff
@@ -56,12 +73,9 @@ func (nrClient *NRClient) Send(buffer *bytes.Buffer, responseChan chan *http.Res
 	}
 	defer resp.Body.Close()
 	defer func() {
-		_, err = io.Copy(ioutil.Discard, resp.Body) // WE READ THE BODY
+		// WE READ THE BODY, err will be returned if there's a problem reading it
```

Comment:
Just a comment for next commits: I would avoid these comments. The method should be self explained.

Reply:
Agree!